packages/discovery-provider/src/tasks/index_tastemaker_notifications.py

```diff
@@ -118,6 +139,7 @@ def create_action_tastemaker_notifications(
         .limit(tastemaker_notification_threshold)
     )
 
+    latest_block = session.query(Block).order_by(desc(Block.number)).first()
```

Comment:
@isaacsolo confirming this is ok. these notifs aren't tied to any particular event so just grabbing latest block. might drift between nodes but since we're not using it for anything i think it's ok. just required to make it [appear as a valid user_challenge here](https://github.com/AudiusProject/audius-protocol/blob/88feea5bcdbc1f17074c65ce5cf6ed18f80d5c7b/packages/discovery-provider/src/challenges/challenge_event_bus.py#L121)

Reply:
yea this is ok